server/src/main/java/com/datastrato/gravitino/server/GravitinoServer.java

```diff
@@ -58,6 +59,13 @@ public GravitinoServer(ServerConfig config) {
   public void initialize() {
     gravitinoEnv.initialize(serverConfig);
 
+    boolean enableAuthorization = serverConfig.get(Configs.ENABLE_AUTHORIZATION);
```

Comment:
I found that this configuration is only used here for checking the validity of service admin configuration.

I think this configuration should control the overall mechanism of AccessControlManager. If this configuration is set to false, I think we don't have to enable access control related mechanism, as well as the related rest endpoints. So please rethink how to leverage this configuration, not just doing a check here.

Reply:
Done.